FluentTerminal.App.ViewModels/TerminalViewModel.cs

```diff
@@ -197,6 +171,12 @@ public TerminalTheme TerminalTheme
             set => Set(ref _terminalTheme, value);
         }
 
+        public OverlayViewModel Overlay
```

Comment:
An AutoProperty should be enough here since this is only set once in the constructor

Reply:
Fix, its now a getter/setter.